spec/javascripts/modules/track-brexit-qa-choices.spec.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-env jasmine, jquery */
+
+var $ = window.jQuery
+
+describe('Brexit QA choices tracker', function () {
+  var GOVUK = window.GOVUK || {}
+  var tracker
+  var $element
+
+  beforeEach(function () {
+    spyOn(GOVUK.SearchAnalytics, 'trackEvent')
+
+    $element = $(
+      '<div>' +
+        '<form onsubmit="event.preventDefault()" data-question-key="question-key">' +
+          '<div>' +
+            '<input name="sector_business_area[]" id="construction" type="checkbox" value="construction">' +
```

Comment:
Minor, but this is out of sync with our input naming.

Reply:
ah yes - shouldn't be an issue, just test data